qubesbuilder/config.py

```diff
@@ -365,7 +363,44 @@ def artifacts_dir(self):
                 self._artifacts_dir = PROJECT_PATH / "artifacts"
         return self._artifacts_dir
 
-    def get_logs_dir(self):
+    @property
+    def temp_dir(self):
+        return self.artifacts_dir / "tmp"
+
+    @property
+    def cache_dir(self):
+        return self.artifacts_dir / "cache"
+
+    @property
+    def sources_dir(self):
+        return self.artifacts_dir / "sources"
+
+    @property
+    def repository_dir(self):
+        return self.artifacts_dir / "repository"
+
+    @property
+    def repository_publish_dir(self):
+        return self.artifacts_dir / "repository-publish"
+
+    @property
+    def distfiles_dir(self):
+        return self.artifacts_dir / "distfiles"
+
+    @property
+    def templates_dir(self):
+        return self.artifacts_dir / "templates"
+
+    @property
+    def installer_dir(self):
+        return self.artifacts_dir / "installer"
+
+    @property
+    def iso_dir(self):
+        return self.artifacts_dir / "iso"
+
+    @property
+    def logs_dir(self):
         return self.artifacts_dir / "logs"
```

Comment:
Old helpers (in the Plugin class) had also `resolve()` call. Is it intentional to skip it here?

Reply:
Yes, self.artifacts_dir is resolved already.